app/page.tsx

```diff
@@ -144,15 +183,35 @@ export default function HomePage() {
   const showPaginationButtons = hasMoreMarkets || page !== 1;
 
   return (
-    <div className="justify-center px-6 mt-12 space-y-12 md:px-10 lg:px-20 xl:px-40 md:flex md:flex-col md:items-center">
+    <div className="justify-center px-6 mt-12 space-y-8 md:px-10 lg:px-20 xl:px-40 md:flex md:flex-col md:items-center">
+      <div className="w-full">
+        <h1 className="text-2xl font-semibold text-white capitalize">
+          🔮 {category ? category : "All"}
```

Comment:
With the previous change with the enum, we can simplify this to use a null coalescing operator

```suggestion
          🔮 {category ?? "All"}
```

Reply:
Sweeeet, thanks!

Reply:
Nono it will only work if you apply all the other changes (use the enum instead the type and default the useState to `useState()` instead of `useState("")`

Reply:
Honestly I would remove this, is just "stealing" space. I would make filters bigger and easier to read instead.

Reply:
@berteotti currently is better with the title. But it will be removed on the next iteration (when we change to correct element) so not important to have it super clean 😁

Reply:
@ElRodrigote I need the empty string. But as I said above it will be removed on the next iteration. So no need to be super clean.